public/js/components/Editor.js

```diff
@@ -271,6 +272,7 @@ const Editor = React.createClass({
     return (
       dom.div(
         { className: "editor-wrapper devtools-monospace" },
+        SourceTabs(),
```

Comment:
Moving the tabs into the editor makes them all part of the same layout flow which makes flexbox easier to layout as otherwise we have two different layouts that work nearly identically. 🙁 Still looking to see if it's avoidable. Also made the `active tab` no longer look like it's part of the editor document unfortunately so I'm still experimenting.